InventoryInStorePickupSales/Model/NotifyOrdersAreReadyForPickup.php

```diff
@@ -97,43 +97,53 @@ public function execute(array $orderIds): ResultInterface
     {
         $failed = [];
         foreach ($orderIds as $orderId) {
-            if (!$this->isOrderReadyForPickup->execute($orderId)) {
-                $failed[] = [
-                    'id' => $orderId,
-                    'message' => 'The order is not ready for pickup'
-                ];
-                continue;
-            }
-
             try {
                 $order = $this->orderRepository->get($orderId);
+
+                $isShipmentCreated = $order->getShipmentsCollection()->getSize() > 0;
```

Comment:
`getShipmentsCollection` is not an API method. Might not work as expected in some customization scenarios. Maybe `\Magento\Sales\Api\ShipmentRepositoryInterface` would work better in this case?